fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflowSpec.groovy

```diff
@@ -56,24 +62,23 @@ class DruidWorkflowSpec extends Specification {
     String uncoveredKey
 
     def setup() {
-        cacheStatus = DRUID_CACHE.isOn()
-        cacheV2Status = DRUID_CACHE_V2.isOn()
         splittingStatus = QUERY_SPLIT.isOn()
 
         systemConfig = SystemConfigProvider.getInstance()
         uncoveredKey = SYSTEM_CONFIG.getPackageVariableName("druid_uncovered_interval_limit")
     }
 
     def cleanup() {
-        DRUID_CACHE.setOn(cacheStatus)
-        DRUID_CACHE_V2.setOn(cacheV2Status)
+        SYSTEM_CONFIG.clearProperty(TTL_CACHE_CONFIG_KEY)
```

Comment:
We need to store the property value in setup, and restore the value in cleanup time so that this test don't cause any side affect to other tests.